tests/comfy_cli/test_workflow_to_api.py

```
@@ -2419,3 +2419,360 @@ def test_batch_option_maps_max_images_and_fail_on_partial(self, seedream_object_
+        assert (values["steps"], values["cfg"], values["sampler_name"]) == (24, 3.5, "ddim")
+
+    def test_count_disagreement_falls_back_to_interior_defaults(self, object_info):
+        """Fail closed. If we cannot say which value belongs to which input,
+        the interior defaults are wrong but self-consistent and executable; a
+        misaligned guess submits a string into an INT slot."""
+        values = self._values(convert_ui_to_api(self._workflow(["two_speakers", "instance prompt"]), object_info))
+        assert values["text"] == "interior text"
+        assert values["mode"] == "interior mode"
+        assert values["steps"] == 99
+
+    def test_count_disagreement_warns(self, object_info, caplog):
```

Pylint (4.0.6)

[warning] Redefining name 'object_info' from outer scope (line 27)

(W0621)